src/keyszer/xorg.py

```diff
@@ -57,7 +57,8 @@ def get_xorg_context():
 
 def get_actual_window(window):
     try:
```

Comment:
This is already inside the try from the outer function, so I think this whole try/except can just be removed as well.

Reply:
The condition isn't guarding for None, try:

```py
        if (wmclass is None and wmname is None) or "FocusProxy" in (wmclass or ""):
```

Reply:
Clever. I knew there had to be a simple way to deal with that. Seems to work. 

Can you do something to clear up the linting error? I keep getting notices about the testing failure after submitting PRs. If there's something I can do to fix it, just let me know.

Reply:
Actually after ever single change I make to any pending PR, I get one of those emails now.